libbeat/outputs/redis/client.go

```diff
@@ -17,6 +18,9 @@ import (
 
 var (
 	versionRegex = regexp.MustCompile(`redis_version:(\d+).(\d+)`)
+
+	ackedEvents    = expvar.NewInt("libbeat.redis.published_and_acked_events")
```

Comment:
I used this name to be similar with `libbeat.elasticsearch.published_and_acked_events`, `libbeat.logstash.published_and_acked_events` and `libbeat.kafka.published_and_acked_events`. I agree that the name is not ideal, so maybe we can change all of them in a separate PR.